packages/playground/remote/src/lib/offline-mode-cache.ts

```diff
@@ -69,11 +74,13 @@ export class OfflineModeCache {
 		 * The development environment uses Vite which doesn't work offline because it dynamically generates assets.
 		 * Check the README for offline development instructions.
 		 */
-		if (
-			url.href.startsWith('http://127.0.0.1:5400/') ||
-			url.href.startsWith('http://localhost:5400/') ||
-			url.pathname.startsWith('/website-server/')
-		) {
+		// @ts-ignore-next-line
+		if (import.meta.env.MODE === 'development') {
+			return false;
+		}
+
+		// Don't cache the local test environment
+		if (url.href.startsWith('https://playground.test/')) {
```

Comment:
Wouldn't that derail the CI runner?

Reply:
The CI runner uses `localhost`, this is why I updated the above check. Playground.test is used by the dev test runner. Without this all e2e broke locally. I would still like to remove this if, but first I need to fix the test.

Reply:
Removed in [7ab4425](https://github.com/WordPress/wordpress-playground/pull/1648/commits/7ab44254bde2e12cccad895e374cdcd3fa91e5fe)